### PR TITLE
All Readthedocs structure and content in place

### PR DESCRIPTION
**Pull request by kbeamnsidc**
This PR should build the docs on RTD...maybe?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -43,5 +43,9 @@ test-watch:
 
 ci: lint test
 
-docs:
+docs-notebook = examples/tutorial.ipynb
+docs/user/notebook.html: $(docs-notebook)
+	jupyter nbconvert --execute --to html --output notebook.html --output-dir docs/user $(docs-notebook)
```
Comment (rwat) [inline Makefile:48]: It's awesome to be able to do this.

File: docs/index.rst [hunk 2]
```diff
@@ -1,19 +1,59 @@
...
+.. image:: https://github.com/nasa/harmony-py/workflows/Python%20package/badge.svg
+    :target: https://github.com/nasa/harmony-py
+
+.. image:: https://readthedocs.org/projects/harmony-py/badge/?version=latest)](https://harmony-py.readthedocs.io/en/latest/?badge=latest
+    :target: https://github.com/nasa/harmony-py
+
+-------------------
+
+**Harmony In Action** ::
+
```
Comment (rwat) [inline docs/index.rst:17]: harmony_client should be instantiated here.

**Comment by rwat** [comment]
When I tried 'make docs' it generated errors around not being able to find `harmony.cli`. I think this was a relic of some previous .rst files but you may want to verify you don't have them too, Kevin.

**Review by rwat — CHANGES_REQUESTED** [review]
Just a few things noted above.

File: .readthedocs.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+sphinx:
+   configuration: docs/conf.py
+
+# Optionally build your docs in additional formats such as PDF
+formats:
+   - pdf
+
+# Optionally set the version of Python and requirements required to build your docs
+python:
+   version: 3.7
```
[the remark this answers is not in the document]
Comment (kbeamnsidc) [reply .readthedocs.yml:18]: For this version, I think it's just the version used when ReadTheDocs builds our documentation after the GitHub webhook, so it should be okay to set this to 3.7.

File: docs/index.rst (hunk 2 above)
Comment (rwat) [reply docs/index.rst:17]: I'm sorry Kevin - you're right.

**Review by bilts — APPROVED** [review]
Nice!

**Pull request merged**